redhat-appstudio/service-provider-integration-operator
# test(pact): add setup for pact contract tests

### What does this PR do?
This PR adds contract tests setup. Once some Pact contract test is saved for SPI in Pact-broker, this setup will run it. Currently, the tests are skipped from unit tests and run separately after unit tests. 

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/HAC-5472

### How to test this PR?
When you run unit tests, e.g. `make test` the pact tests should be run as part of this suite. You can run unit tests separately by `make unit-tests` or pact tests by `make pact-tests`.

```
quay.io/redhat-appstudio/pull-request-builds:spi-controller-#
quay.io/redhat-appstudio/pull-request-builds:spi-oauth-#
```
